### DISCUSSION TITLE
Various genres not displaying servers

### DISCUSSION THREAD

**Discussion by rdica** [jamulussoftware/jamulus#3301] [Technical Support & Help] [answered]
Are there ongoing issues with the directory servers for Rock, Jazz, Classical, and Choral genres?
None of these return any data in the Jamulus client or https://explorer.jamulus.io

**Comment by rdica** [comment]
Checking back on some things, it appears the issue started around 2024-06-26-13:31:19+0000 and lasted til around 2024-06-27-02:10:00+0000 after which i could see servers listed in the affected genres in client and https://explorer.jamulus.io

**Comment by ann0see** [comment]
@pljones should know

**Comment by pljones** [comment, marked as answer]
> Are there ongoing issues with the directory servers for Rock, Jazz, Classical, and Choral genres? None of these return any data in the Jamulus client or https://explorer.jamulus.io

Our ISP had an outage and, as I don't use mobile data very often, all communications were cut as I was getting challenge emails sent to email services that I could only access by typing in a code sent to a service I didn't have access to, due to internet outage!  We were without internet from around 2.30pm to 3.10am BST on Wednesday-Thursday.

(When contacted, our ISP said they said they weren't having any problems at all but there was a problem on our property.  Miraculously, it cleared up overnight whilst we slept...)

**Reply by rdica** [reply]
Thanks for the update @pljones glad everythings working, cheers!
